### ISSUE TITLE
WCAG violation: scrollable element is not operable through a keyboard

### ISSUE THREAD

**Issue by KjerstiSteien** [zazuko/Yasgui#40]
According to WCAG 2.1.1 and 2.1.3: All functionality of the content is operable through a keyboard without requiring specific timings for individual keystrokes.

https://www.w3.org/WAI/WCAG22/Understanding/keyboard
https://www.w3.org/WAI/WCAG22/Understanding/keyboard-no-exception

The scrollable element in the SPARQL GUI is not available through a keyboard.
